### ISSUE TITLE
aria-valuemax with stepsArray has array length instead of value

### ISSUE THREAD

**Issue by s-kirchner** [angular-slider/ngx-slider#403]
When a stepsArray is used, the aria-valuemin and aria-valuemax values are showing array sizes, and not the corresponding values. Taking the example from the official homepage:

`import { Component } from '@angular/core';
import { Options } from '@angular-slider/ngx-slider';

@Component({
    selector: 'app-custom-ticks-legend-slider',
    templateUrl: './custom-ticks-legend-slider.component.html',
    standalone: false
})
export class CustomTicksLegendSliderComponent {
  value: number = 5;
  options: Options = {
    showTicksValues: true,
    stepsArray: [
      {value: 1, legend: 'Very poor'},
      {value: 2},
      {value: 3, legend: 'Fair'},
      {value: 4},
      {value: 5, legend: 'Average'},
      {value: 6},
      {value: 7, legend: 'Good'},
      {value: 8},
      {value: 9, legend: 'Excellent'}
    ]
  };
}`

This renders the following HTML code:
`<span _ngcontent-ng-c4030736784="" ngxsliderhandle="" class="ngx-slider-span ngx-slider-pointer ngx-slider-pointer-min" style="opacity: 1; visibility: visible; left: 615px; transform: rotate(0deg);" role="slider" tabindex="0" aria-orientation="horizontal" aria-label="ngx-slider" aria-labelledby="" aria-valuenow="5" aria-valuetext="5" aria-valuemin="0" aria-valuemax="8"></span>`

aria-valuemin=0 and aria-valuemax=8 but they should be 1 and 9 respectively

**Comment by AlDrag** [comment]
I can see the issue. Theoretically not a difficult fix 🤞

I do wonder what the `aria-valuemin` and `aria-valuemax` values should be when the `bindIndexForStepsArray` option is enabled. Possibly should use the index as the value.
